src/toil/batchSystems/abstractBatchSystem.py

```diff
@@ -170,6 +170,82 @@ def workerCleanup(workerCleanupInfo):
         # TODO: logic here for cleanup flag
         shutil.rmtree(workflowDir)
 
+    @staticmethod
+    def probeUsedDisk(workflowID, workDir=None):
+        '''
+        Used to probe the used disk space (on the disk mounting workDir) minus the space used by
```

Comment:
"on the disk mounting workDir" should be "on the block device containing workDir" or "on the block device containing the working directory (workDir)"